src/main/kotlin/de/rwth_erstis/discordbot_jvm/Launcher.kt

```diff
@@ -11,4 +12,6 @@ fun main(args: Array<String>) {
     val env = dotenv()
     val token = env["TOKEN"] ?: exitProcess(1)
     val bot = Bot(token)
+    val cmdHandler = CommandHandler(bot)
+    cmdHandler.registerCommand(Help())
```

Comment:
Registering of commands should be done in the cmdHandlers Constructor.

Optionally you can register them in a "init{ }" block in the BotImpl but i dont really know what the benefits of this would be.